### ISSUE TITLE
Support for simple IN operator

### ISSUE THREAD

**Issue by javajack** [eaxdev/JsonSQL4J#5]
Do you have any plan to support simple IN operator ?
I see that this library is still a work in progress.

Have you considered adding IN operator support ? 
A simple IN operator dealing with string,long,double values would be a great addition.

example,

IN('US','UK','IN','NZ')

IN(1,2,3)

**Comment by eaxdev** [comment]
Yes, this is a good idea. I'll do it soon

**Comment by eaxdev** [comment]
Implemented in the version 0.0.3. You can use json like this:
```json
{
  "fields": [
    {
      "column": "field1"
    },
    {
      "column": "field2",
      "alias": "test"
    }
  ],
  "from": [
    {
      "table": "table1",
      "schema": "schema"
    }
  ],
  "where": {
    "or": [
      {
        "in": {
          "fieldName": "field3",
          "values": [
            "US",
            "UK",
            "IN",
            "NZ"
          ]
        }
      },
      {
        "eq": {
          "fieldName": "field4",
          "value": "3"
        }
      }
    ]
  }
}

```
